docs/weighting/poiAndTransitNodeWeighting_en.md

```diff
@@ -0,0 +1,346 @@
+- $b$ = power parameter ($> 0$)
+- $c$ = exponential decay parameter ($> 0$)
+
+**Typical coefficient ranges by trip purpose (Medium MPO):**
+
+| Trip Purpose | a | b | c | Application |
+|--------------|---|---|---|-------------|
+| Home-Based Work | 2000-6000 | 0.8-1.2 | 0.05-0.10 | Work commute trips |
+| Home-Based Non-Work | 1500-3500 | 0.5-0.9 | 0.08-0.15 | Shopping, social, personal |
+| Non-Home-Based | 1000-2500 | 0.4-0.8 | 0.10-0.18 | Chained trips |
+
+**Example calculation:** For home-based work trips with $t=15$ minutes using parameters $a=5280$, $b=0.926$, $c=0.087$:
+
+$$\begin{align*}
+f(15) &= 5280 \times 15^{-0.926} \times \exp(-0.087 \times 15) \\
+      &= 5280 \times 0.0748 \times 0.267 \\
+      &= 105.4 \text{ (friction factor)}
+\end{align*}$$
+
+## Distance and time decay parameters across modes and purposes
+
+Empirical studies show $\beta$ values vary dramatically by mode and trip purpose, quantifying that pedestrians are extremely distance-sensitive while drivers tolerate longer trips.
+
+### Distance-based parameters
+
+| Mode/Purpose | Distance Decay ($\beta$) | Function Type | Data Source |
+|--------------|-------------------|---------------|-------------|
+| **Walking** |
+| - Work | 1.683 | Power | Iacono et al. (2008) |
+| - Shopping | 2.106 | Power | Iacono et al. (2008) |
+| - Entertainment | 0.769 | Power | Iacono et al. (2008) |
+| - To Transit (network) | 0.00217 | Exponential | Bartzokas-Tsiompras (2019) |
+| **Transit** |
+| - Local Bus | 0.186 | Power | Iacono et al. (2008) |
+| - Light Rail | 0.046 | Power | Iacono et al. (2008) |
+| - Express Bus | 0.060 | Power | Iacono et al. (2008) |
+| **Auto** |
+| - Work | 0.09 | Power | Iacono et al. (2008) |
+| - Shopping | 0.118 | Power | Iacono et al. (2008) |
+
+### Travel time-based parameters
+
+| Mode | $\beta$ (time-based) | Function Type | Interpretation |
+|------|----------------|---------------|----------------|
+| Walking | 0.20-0.30 | Exponential | Very steep decay (high time sensitivity) |
+| Local transit | 0.08-0.12 | Exponential | Moderate decay |
+| Auto | 0.03-0.06 | Exponential | Shallow decay (low time sensitivity) |
+
+**For accessibility studies:**
+- Employment access: $\beta = 0.08\text{-}0.12$
+- Retail access: $\beta = 0.15\text{-}0.20$
+- Healthcare access: $\beta = 0.10\text{-}0.15$
+
+### When to use travel time vs. distance
+
+**Use travel time when:**
+1. Multi-modal analysis (transit vs. auto have different time-distance relationships)
+2. Congested urban networks (speed varies significantly)
+3. Accessibility to jobs/services (time is better proxy for opportunity cost)
+4. Transit catchment areas with varied walking speeds
+5. Four-step travel demand modeling (NCHRP 716 standard)
+
+Transition uses travel time output from the integrated routing engines for walking, cycling, transit and driving, but can also use network distance or Euclidean/bird's-eye distance when routing engines are not available or when testing.
```

Comment:
is bird's-eye the right expression in english? I also hear often as the crow fly